packages/gatsby-core-utils/README.md

```diff
@@ -63,3 +63,30 @@ const pathname = "./gatsby/is/awesome"
 const url = joinPath(BASEPATH, pathname)
 // ...
 ```
+
+### isCI
```

Comment:
We love it when folks update documentation! 🥇 for that! 🤗

Reply:
My pleasure :)